### ISSUE TITLE
[FEAT] MAIN About

### ISSUE THREAD

**Issue by dudlyart** [Redlion-news/issues#24]
**Is your feature request related to a problem? Please describe.**
I can't find About page content in CMS, is it editable? 
![image](https://user-images.githubusercontent.com/81701877/198099436-c3c27e6d-2491-4b8f-b5ec-a8b2dd7e2255.png)

**Comment by qruz-hq** [comment]
Yup it is editable but i used a private contentful space to test out things, will merge it into the current production during the day
